src/styles/blocks/header.scss

```diff
@@ -0,0 +1,72 @@
+/* #region Header */
+.header {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  align-items: center;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  border-radius: 8px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin-left: 50px;
```

Comment:
Don't use external styles (e,g, margin) for blocks.
Make it an element of `header` block or use BEM mix.
https://www.youtube.com/watch?v=JOJ-mENrg_s&t=1s&ab_channel=Mateacademy